packages/fxa-auth-server/lib/senders/emails/templates/subscriptionFailedPaymentsCancellation/en.ftl

```diff
@@ -0,0 +1,7 @@
+#  Variables:
+#  $productName (String) - The name of the subscribed product, e.g. Mozilla VPN
+subscriptionFailedPaymentsCancellation-title = Your subscription has been cancelled
+subscriptionFailedPaymentsCancellation-subject = Your <%- productName %> subscription has been cancelled
```

Comment:
An EJS tag sneaked by, the variable should be `{ $productName }`.

Nitpick: The other `ftl` files have the email subject line before the title.